### PR TITLE
WIP: chore(RHIDP-4037): Compare, 1.3

### PR DESCRIPTION
**Pull request by jhutar**

### CODE REVIEW

File: test.env [hunk 1]
```diff
@@ -56,3 +56,12 @@
 # export ARTIFACT_DIR=.artifacts
 # export ENABLE_RBAC=false
 # export ENABLE_PROFILING=false
+
+export DURATION=5m
+export PRE_LOAD_DB=true
+export SCENARIO=mvp
+export USE_PR_BRANCH=true
+export WAIT_FOR_SEARCH_INDEX=true
```
Comment (pmacik) [inline test.env:64]: @jhutar that should stay false until https://issues.redhat.com/browse/RHIDP-2247 is finished

Comment (jhutar) [reply test.env]: Thank you, fixed.

**Comment by jhutar** [comment]
Tests done, closing PR.

**Pull request closed without merging**